test/bases/renku_data_services/data_api/test_projects.py

```diff
@@ -1247,7 +1204,7 @@ async def test_project_copy_basics(
     assert copy_project == snapshot(exclude=props("id", "updated_at", "creation_date", "etag", "template_id"))
 
     _, response = await sanic_client.get(
-        f"/api/data/projects/{copy_project["id"]}", params={"with_documentation": True}, headers=user_headers
+        f"/api/data/projects/{copy_project['id']}", params={"with_documentation": True}, headers=user_headers
```

Comment:
Are these single quotes on purpose here?

Reply:
Ah that is my autoformatting doing weird things. I will remove that.